File: replace.py
import re

origin = """"""

step1 = re.sub("(\d+\.)", "\g<1> ", origin)
step2 = re.sub("(\d+)、", "\g<1>. ", step1)
step3 = re.sub("-", "", step2)
step4 = re.sub("(\d+\. \S+)", "**\g<1>**", step3)

step5 = """
<details>
<summary>【虎扑步行街20个简短而深刻的亮回复】第 期！</summary>\n""" + step4 + "\n</details>"

print(step5)

File: replace.js
var origin = ``

var res = origin
    .replace(/(\d+\.)/g, "$1")
    .replace(/(\d+)、/g, "$1. ")
    .replace(/-/g, "")
    .replace(/(\d+\. \S+)/g, "**$1**")
    .replace(/  /g, " ")

res = `
<details>
<summary>【虎扑步行街20个简短而深刻的亮回复】第 X 期！</summary>\n` + res + "\n\n</details>"

console.log(res)
